core/src/main/scala/kafka/server/ReplicaManager.scala

```diff
@@ -74,8 +74,7 @@ case class LogReadResult(info: FetchDataInfo,
   }
 
   override def toString =
-    s"Fetch Data: [$info], HW: [$hw], leaderLogEndOffset: [$leaderLogEndOffset], readSize: [$readSize], error: [$error]"
-
+    s"Fetch Data: [$info], HW: [$hw], leaderLogEndOffset: [$leaderLogEndOffset], readSize: [$readSize], error: [${error.exceptionName}]"
```

Comment:
I noticed we are using the exception name in other places (like [here](https://github.com/apache/kafka/blob/9898d665d1ab201405d66c70e3ea9710d9dcecd7/core/src/main/scala/kafka/api/OffsetResponse.scala#L50), or [here](https://github.com/apache/kafka/blob/9898d665d1ab201405d66c70e3ea9710d9dcecd7/core/src/main/scala/kafka/api/TopicMetadata.scala#L77)). Plus, to me the Errors name is more for internal use, and the exception name is what should be exposed to users. Thanks.